handler/handler.go

```diff
@@ -0,0 +1,148 @@
+package handler
```

Comment:
rename `handler` to `http/server` or `transport/http/server` because it runs HTTP server when a handler is just an implementation of `http.Handler`